google/ggrc-core
# GGRC-5323 Auditor can't update document mapped to issue

# Issue description

There need to add update right to document in issue for auditor.

# Steps to test the changes

1. Login as admin.
2. Create audit.
3. Add other user `[2]` as auditor in audit.
4. Add issue to audit.
5. Add document to issue.
6. Login as user `[2]`.
7. Check that you can edit this document.

# Solution description

Write migration for changing access control roles propagation tree.

# Sanity checklist



# Migration checklist


# PR Review checklist
